src/audiomixerboard.cpp

```diff
@@ -64,6 +66,11 @@ CChannelFader::CChannelFader ( QWidget*     pNW,
     pFader->setRange        ( 0, AUD_MIX_FADER_MAX );
     pFader->setTickInterval ( AUD_MIX_FADER_MAX / 9 );
 
+    // setup panning slider
+    pPan->setRange( 0, AUD_MIX_FADER_MAX);
```

Comment:
The problem is that if the user changed the pan, he will never be able to set it back exactly in the middle again. We would need some sort of "snap to the middle" functionality.
Also, have you checked how the new slider looks in "fancy" GUI style? Does it show up nicely?

Reply:
Yes, this is not implemented yet. A user friendly way would be maybe  to have doubleclick on the slider handle to put it to default position (center)  but this requires subclassing QSlider and reimplementing its mouseDoubleClickEvent  https://doc.qt.io/qt-5/qwidget.html#mouseDoubleClickEvent -  or adding a small button to center it. 

In practice I don't think it is very crucial, if the pan is set to the middle by hand more or less, it sounds in center enough.

I did not do any work yet on the styleSheet or look and feel, this is now just to test the functionality.

Reply:
Sorry for the delay. Now I again have time for your pull request :-).
I do not find a double-click very intuitive. Let's keep the issue with the centering open for now.